library/globals.inc.php

```diff
@@ -831,13 +831,6 @@ function gblTimeZones()
             xl('Set a facility cookie to remember the selected facility between logins.')
         ),
 
-        'login_into_facility' => array(
-            xl('Login Into Facility'),
-            'bool',                           // data type
-            '0',                              // default
-            xl('Select your current facility in the login page')
-        ),
-
```

Comment:
revert this back

Reply:
nuts but then again, this doesn't make sense! What use is this when even patient are not linked to facilities except through provider which is only useful in billing. Is someone going to add the business logic where this make sense or is this custom for a vendor or perhaps institutional(we can hope).